src/runtime/process.rs

```diff
@@ -549,3 +555,37 @@ impl Default for LinuxSchedulerFlag {
         LinuxSchedulerFlag::SchedResetOnFork
     }
 }
+
+#[derive(Builder, Clone, Debug, Deserialize, Getters, Setters, Eq, PartialEq, Serialize)]
+#[builder(
+    default,
+    pattern = "owned",
+    setter(into, strip_option),
+    build_fn(error = "OciSpecError")
+)]
+#[getset(get = "pub", set = "pub")]
+/// ExecCPUAffinity specifies CPU affinity used to execute the process.
+/// This setting is not applicable to the container's init process.
+pub struct ExecCPUAffinity {
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    /// cpu_affinity_initial is a list of CPUs a runtime parent process to be run on
+    /// initially, before the transition to container's cgroup.
+    /// This is a a comma-separated list, with dashes to represent ranges.
+    /// For example, `0-3,7` represents CPUs 0,1,2,3, and 7.
+    cpu_affinity_initial: Option<String>,
+
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    /// cpu_affinity_final is a list of CPUs the process will be run on after the transition
+    /// to container's cgroup. The format is the same as for `initial`. If omitted or empty,
+    /// the container's default CPU affinity, as defined by cpu.cpus property, is used.
+    cpu_affinity_final: Option<String>,
```

Comment:
Originally fields in `execCPUAffinity` are `initial` and `final`.

https://github.com/opencontainers/runtime-spec/blob/701738418b9555d5213337a0991fd0ffd6c37808/config.md?plain=1#L343-L353

Since `final` is a reserved keyword in Rust, I added `cpu_affinity_` as prefix.